rotkehlchen/exchanges/bitfinex.py

```diff
@@ -661,7 +658,13 @@ def _query_currency_map(self) -> CurrencyMapResponse:
                     bfx_symbol: symbol for bfx_symbol, symbol in response_list[0]
                     if bfx_symbol not in set(BITFINEX_EXCHANGE_TEST_ASSETS)
                 }
-                currency_map.update(BITFINEX_TO_WORLD)
+                with GlobalDBHandler().conn.read_ctx() as cursor:
+                    cursor.execute(
+                        'SELECT value, local_id FROM location_asset_mappings WHERE location IS ? OR location IS NULL;',  # noqa: E501
+                        (Location.BITFINEX.serialize_for_db(),),
+                    )
+                    for bfx_symbol, asset in cursor:
+                        currency_map[bfx_symbol] = asset
```

Comment:
To avoid possible errors in the future

```suggestion
                    for bfx_symbol, asset_id in cursor:
                        currency_map[bfx_symbol] = asset_id
```